### PR TITLE
Tweaks

### PR DESCRIPTION
**Pull request by rpassas**
# Summary
- updated tables to be a bit more readable
- updated nav bar so that drop down is more visually discriminated and arrow is clickable/visible when pressed
- fixed links
- added links

### CODE REVIEW

**Comment by arscan** [comment]
Hmmm, I like how you made the arrows clickable but I think that the way the sub-elements are highlighted (now in light grey) when the parent is selected is ugly.  I understand why it works like that technically, but to me it looks very strange.

Screenshot 2024-01-31 at 3 30 16 PM

File: docs/concepts.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ or reference implementations.
 
 ## Inferno Template
 
-A template for writing Inferno Test Kits.
+A template for writing Inferno Test Kits. See [Template Layout](http://localhost:4000/docs/getting-started/repo-layout-and-organization) for more information.
```
Comment (arscan) [inline docs/concepts.md:35]: Shouldn't be 'localhost'.  Just make it a relative link.

Comment (rpassas) [reply docs/concepts.md:35]: I think there are some other `localhost` links throughout the site. I assumed that was on purpose. I can go through and make them all relative.

Comment (arscan) [reply docs/concepts.md:35]: Nope, just a mistake!

**Comment by rpassas** [comment]
> Hmmm, I like how you made the arrows clickable but I think that the way the sub-elements are highlighted (now in light grey) when the parent is selected is ugly. I understand why it works like that technically, but to me it looks very strange.
> Screenshot 2024-01-31 at 3 30 16 PM

Can remove the graying of the children. I tried this since we had talked about making it clearer to the user what the sub-pages are. This seemed like a simpler way of doing that than adding a table of contents to the parent page.

**Review by arscan — APPROVED** [review]

**Pull request merged**